marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployDatabaseFieldCommand.java

```diff
@@ -61,6 +61,11 @@ protected String adjustPayloadBeforeSavingResource(CommandContext context, File
         return addExistingFieldsAndRangeFieldIndexes(payload, new DatabaseManager(context.getManageClient()));
     }
 
+    @Override
+    public void undo(CommandContext context) {
+        //no-op
```

Comment:
I should have mentioned this before - I recommend a log statement here just to answer "why" this method is overridden to not do anything - e.g. logger.info("No action required on undeploy, as the command for deleting databases on undeploy will also delete the fields and indexes created by this command.");

Reply:
A comment too, in the code, would be nice so we know why that is a no-op in case we need to change it for whatever reason. (or just a link to the jira ticket #)